templates/pro/free-trial.html

```diff
@@ -0,0 +1,369 @@
+{% from "_macros/vf_hero.jinja" import vf_hero %}
+
+{% extends "templates/base.html" %}
+
+{% block title %}Ubuntu Pro Free trial{% endblock %}
+
+{% block meta_copydoc %}
+  https://docs.google.com/document/d/1USgQJcUKNz3iZYeNVeICVqGz879rlG5ztEtVxPWm8bc
+{% endblock
+meta_copydoc %}
+
+{% block meta_description %}
+  Ubuntu Pro is a single, comprehensive subscription for open source security and compliance. Try it out for 30 days and see first-hand how your organisation can benefit from it.
+{% endblock meta_description %}
+
+{% block body_class %}
+  is-paper
+{% endblock body_class %}
+
+{% macro accordion(prefix, items, is_first_open=False ) %}
+  <aside class="p-accordion">
+    <ul class="p-accordion__list">
+      {% for title, description in items %}
+        {% set unique_id = prefix ~ loop.index0 %}
+        <li class="p-accordion__group">
+          <strong role="heading" aria-level="4" class="p-accordion__heading">
+            <button type="button"
+                    class="p-accordion__tab"
+                    id="{{ unique_id }}-tab"
+                    aria-controls="{{ unique_id }}-tab-section"
+                    aria-expanded="{{ 'true' if loop.first and is_first_open else 'false' }}">{{ title }}</button>
+          </strong>
+          <section class="p-accordion__panel"
+                   id="{{ unique_id }}-tab-section"
+                   aria-hidden="{{ 'false' if loop.first and is_first_open else 'true' }}"
+                   aria-labelledby="{{ unique_id }}-tab">
+            <p>{{ description | safe }}</p>
+          </section>
+        </li>
+      {% endfor %}
+    </ul>
+  </aside>
+{% endmacro %}
+
+{% block outer_content %}
+
+  {% block content %}
+    {% call(slot) vf_hero(
+      title_text='Ubuntu Pro Free Trial',
+      layout='50/50'
+      ) -%}
+      {%- if slot == 'description' -%}
+        <p>
+          Ubuntu Pro is a single, comprehensive subscription for open source security and compliance. Try it out for 30 days and see first-hand how your organisation can benefit from expanded security maintenance, Kernel Livepatching, and automations for device management and compliance.
+        </p>
+      {%- endif -%}
+      {%- if slot == 'cta' -%}
+        <a href="#" class="p-button--positive">Discover Ubuntu Pro</a>
+      {%- endif -%}
+      {%- if slot == 'image' -%}
+        {{ image(url="https://assets.ubuntu.com/v1/dcae3633-hero-image.jpg",
+                alt="",
+                width="1200",
+                height="752",
+                hi_def=True,
+                loading="auto",
+                attrs={"class": "u-hide--medium u-hide--small"}) | safe
+        }}
+      {%- endif -%}
+    {% endcall -%}
+
+    <section class="p-section">
```

Comment:
We can use the [tiered list pattern](https://vanillaframework.io/docs/patterns/tiered-list#5050-with-description) here

Reply:
I didn't know, thanks